rubygem-smart_proxy_dynflow_core/rubygem-smart_proxy_dynflow_core.spec

```diff
@@ -0,0 +1,142 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%if 0%{?scl:1}
+    %global bin_dir %{_root_bindir}
+%else
+    %global bin_dir %{_bindir}
+%endif
+
+%global gem_name smart_proxy_dynflow_core
+%global service_name smart_proxy_dynflow_core
+
+Summary: Core Smart Proxy Dynflow Service
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.1.1
+Release: 1%{?foremandist}%{?dist}
+Group: Development/Libraries
+License: GPLv3
+URL: http://github.com/theforeman/smart_proxy_dynflow
+Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
+Requires: foreman-proxy >= 1.11.0
+
+Requires: %{?scl_prefix}rubygem(bundler_ext)
+Requires: %{?scl_prefix}rubygem(dynflow) >= 0.8.4
+Requires: %{?scl_prefix}rubygem(dynflow) < 0.9.0
+Requires: %{?scl_prefix}rubygem(sequel)
+Requires: %{?scl_prefix}rubygem(rest-client)
+Requires: %{?scl_prefix_ror}rubygem(sinatra) >= 1.4
+Requires: %{?scl_prefix_ror}rubygem(sinatra) < 2.0
+Requires: %{?scl_prefix_ror}rubygem(rack)
+Requires: %{?scl_prefix_ror}rubygem(sqlite3)
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}rubygems
+%if 0%{?rhel} == 6
+Requires(post): chkconfig
+Requires(preun): chkconfig
+Requires(preun): initscripts
+%else
+Requires(post): systemd-sysv
+Requires(post): systemd-units
+Requires(preun): systemd-units
+BuildRequires: systemd
+%endif
+
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}rubygems
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+
+%description
+Use the Dynflow inside Foreman smart proxy
+
+%package doc
+BuildArch:  noarch
+Requires:   %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+Summary:    Documentation for rubygem-%{gem_name}
+
+%description doc
+This package contains documentation for rubygem-%{gem_name}.
+
+%prep
+%setup -n %{pkg_name}-%{version} -q -c -T
+mkdir -p .%{gem_dir}
+%{?scl:scl enable %{scl} "}
+gem install --local --install-dir .%{gem_dir} \
+            --force %{SOURCE0} --no-rdoc --no-ri
+%{?scl:"}
+
+%build
+sed -ri '1sX(/usr/bin/ruby|/usr/bin/env ruby)X/usr/bin/%{?scl:%{scl_prefix}}rubyX' %{_builddir}/%{pkg_name}-%{version}/%{gem_instdir}/bin/%{service_name}
+
+# switches to bundler_ext instead of bundler and uses bundler.d from /etc/foreman-proxy/dynflow/bundler.d
+mv %{_builddir}/%{pkg_name}-%{version}/%{gem_instdir}/Gemfile %{_builddir}/%{pkg_name}-%{version}/%{gem_instdir}/Gemfile.in
+sed -ri 'sX\#\{File\.dirname\(__FILE__\)\}/bundler\.d/\*\.rbX%{_sysconfdir}/foreman-proxy/dynflow/bundler.d/*.rbX' %{_builddir}/%{pkg_name}-%{version}/%{gem_instdir}/Gemfile.in
+
+%install
+mkdir -p %{buildroot}%{_sysconfdir}/foreman-proxy/dynflow/bundler.d
```

Comment:
Would it make sense to use datadir and only store the bunder.d there but keep the ruby code in the gem dir? Foreman and smart proxy seem to get it all in the same place.